hera_mc/tests/__init__.py

```diff
@@ -15,13 +15,17 @@
 import pytest
 
 from hera_mc.utils import get_iterable
+TEST_DEFAULT_REDIS_HOST = None
```

Comment:
This is just weird. Why define a variable like this and set it to None?

Reply:
This allows you to set it to a different host if you are testing locally.

Reply:
This then keeps it consistent throughout testing, which is what you want.

Reply:
I personally still find this a bit hacky. I would prefer if developers had a redis server that lives at "redishost" on their machine. Adding a docker container running redis to a developers host file as "redishost" is pretty simple to execute 

As a second option, we could add a command line argument to the testing setup that will allow users to feed in a custom host for redis when they execute tests. 

I am fine with either option.

Reply:
As it currently is, you nor any other developer needs to change anything when testing if they want to spin up 'redishost' or have no redis server.  I think that having a default host (even if None) set in one place that works for users is fine.  Having someone need to run a command line before testing seems hacky-er to me.

Reply:
I really don't understand this approach. It hardcodes a new variable to `None`. I can't see any benefit from that, nor do I see how it allows someone running pytest on their machine to set a different host for their local machine (which Matt's suggestion would allow).

Reply:
How about if we move along on this PR and make an issue to add a command line option to the test redis hostname.  I'm trying to juggle this with the hera_corr_f update.

Reply:
Fixed via zoom meeting